client/src/components/MultiJurisdictionAudit/RoundManagement/index.tsx

```diff
@@ -56,22 +61,27 @@ const RoundManagement = ({ round, auditBoards, createAuditBoards }: IProps) => {
     auditBoards,
   ])
 
-  // const [{ online }] = useAuditSettings(electionId!)
+  const settings = useAuditSettingsJurisdictionAdmin(electionId, jurisdictionId)
+
+  const online = settings ? settings.online : false
```

Comment:
maybe `return null` if settings are still loading (like we do with ballots below)?

Reply:
I changed it to do something similar.